packages/lab/src/input-next/InputNext.tsx

```diff
@@ -196,13 +153,17 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         className={clsx(withBaseName("input"), inputProps?.className)}
         disabled={isDisabled}
         readOnly={isReadOnly}
-        ref={ref}
         value={value}
         tabIndex={isReadOnly || isDisabled ? -1 : 0}
         onBlur={handleBlur}
         onChange={handleChange}
         onFocus={!isDisabled ? handleFocus : undefined}
+        {...a11yProps}
         {...restInputProps}
+        aria-labelledby={clsx(
+          a11yProps?.["aria-labelledby"],
```

Comment:
Spreading `a11yProps` first and then picking one key out to override may lead to code confusion. e.g. we destructure things we need to modify from `props` and spread rest